{
  "schema_version": "1.5.0",
  "id": "BIT-python-2022-48564",
  "details": "read_ints in plistlib.py in Python through 3.9.1 is vulnerable to a potential DoS attack via CPU and RAM exhaustion when processing malformed Apple Property List files in binary format.",
  "aliases": [
    "CVE-2022-48564"
  ],
  "affected": [
    {
      "package": {
        "ecosystem": "Bitnami",
        "name": "python",
        "purl": "pkg:bitnami/python"
      },
      "severity": [
        {
          "type": "CVSS_V3",
          "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H"
        }
      ],
      "ranges": [
        {
          "type": "SEMVER",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "3.6.13"
            },
            {
              "introduced": "3.7.0"
            },
            {
              "fixed": "3.7.10"
            },
            {
              "introduced": "3.8.0"
            },
            {
              "fixed": "3.8.7"
            },
            {
              "introduced": "3.9.0"
            },
            {
              "fixed": "3.9.1"
            }
          ]
        }
      ]
    }
  ],
  "database_specific": {
    "severity": "Medium",
    "cpes": [
      "cpe:2.3:a:python:python:*:*:*:*:*:*:*:*"
    ]
  },
  "references": [
    {
      "type": "WEB",
      "url": "https://bugs.python.org/issue42103"
    },
    {
      "type": "WEB",
      "url": "https://lists.debian.org/debian-lts-announce/2023/10/msg00017.html"
    },
    {
      "type": "WEB",
      "url": "https://security.netapp.com/advisory/ntap-20230929-0009/"
    }
  ],
  "published": "2024-01-31T15:23:59.380Z",
  "modified": "2024-01-31T15:40:39.817Z"
}